src/main/java/com/android/volley/toolbox/HurlStack.java

```diff
@@ -41,13 +41,7 @@
     private static final int HTTP_CONTINUE = 100;
 
     /** An interface for transforming URLs before use. */
-    public interface UrlRewriter {
-        /**
-         * Returns a URL to use instead of the provided one, or null to indicate this URL should not
-         * be used at all.
-         */
-        String rewriteUrl(String originalUrl);
-    }
+    public interface UrlRewriter extends com.android.volley.toolbox.UrlRewriter {}
```

Comment:
Just for my own understanding, what is this for? It extends the toolbox.UrlRewriter, but doesn't seem to add any methods

Reply:
I didn't want to break the API, since HurlStack could take in a HurlStack.UrlRewriter.